crates/swc_ecma_minifier/src/pass/mangle_names/preserver.rs

```diff
@@ -24,6 +24,12 @@ pub(super) struct Preserver {
     in_top_level: bool,
 }
 
+impl Preserver {
+    fn is_reserved(&self, ident: &Ident) -> bool {
+        self.options.reserved.contains(&ident.sym.to_string())
```

Comment:
`&ident.sym.to_string()` needlessly allocates and drops the memory right away.
This will be fixed if you change type to the `JsWord`

Reply:
resolved in c12701c9208b636d660ef23458d9aaf87706d2bc